libs/perun/components/src/lib/perun-footer/perun-footer.component.html

```diff
@@ -62,13 +62,21 @@ <h6 [ngStyle]="{'color': headersTextColor}" class="font-weight-bold">
         >{{index ? ', ' + item.name : item.name}}</a
       >
     </span>
-    Version
+    , GUI: v.
     <a
       [href]="githubRepository"
       [ngStyle]="{'color': copyrightTextColor}"
       class="text-decoration-none"
       target="_blank"
-      >{{version}}</a
+      >{{guiVersion}}</a
+    >
+    , Backend: v.
```

Comment:
There are suggestions like `Backend`,  `RPC` or `core`, so I also would like to suggest my option. I am not sure what is commonly used across applications, but the `server` is used at least in our old GUI, so our users should be familiar with it.
```suggestion
    , server: v.
```